### ISSUE TITLE
expire command end: aborted with exception [061]

### ISSUE THREAD

**Issue by hunleyd** [pgbackrest/pgbackrest#560]
(opening an issue as per David on slack)

This is backrest 2.04 on OSX 10.13.6 compiled from a tarball.

My .conf:
```
[10]
db-path=/usr/local/var/postgres/10
db-port=5410
repo1-path=/Volumes/Google/Backups/db/10

[global]
log-level-console=info
process-max=4
repo1-type=cifs
resume=n
retention-full=2
start-fast=y
```

The `/Volumes/Google` mount is a FUSE mount of my Google One storage. Whenever I run a `full` backup, I'm seeing:
```
2018-07-16 11:31:50.294 P04   INFO: backup file /usr/local/var/postgres/10/pg_logical/replorigin_checkpoint (8B, 100%) checksum 347fc8f2df71bd4436e38bd1516ccd7ea0d46532
2018-07-16 11:31:50.332 P00   INFO: full backup size = 202.6MB
2018-07-16 11:31:50.332 P00   INFO: execute non-exclusive pg_stop_backup() and wait for all WAL segments to archive
2018-07-16 11:31:50.506 P00   INFO: backup stop archive = 0000000200000000000000DE, lsn = 0/DE000130
2018-07-16 11:31:51.330 P00   INFO: new backup label = 20180716-113101F
2018-07-16 11:31:51.401 P00   INFO: backup command end: completed successfully (50357ms)
2018-07-16 11:31:51.401 P00   INFO: expire command begin
2018-07-16 11:31:51.426 P00   INFO: expire full backup 20180716-093105F
2018-07-16 11:31:51.481 P00   INFO: remove expired backup 20180716-093105F
2018-07-16 11:32:57.145 P00   INFO: expire command end: aborted with exception [061]
```
The expired backup *is* removed successfully though.

**Comment by dwsteele** [comment]
This is essentially an object store under the covers, so I wonder if directories are just going away once the files are deleted and causing errors.

First, though, I would like to figure out why no error message is being output.

**Comment by hunleyd** [comment]
Let me know if there's something needed on my end (debug/trace output)

**Comment by dwsteele** [comment]
I figured out why the error message is not getting logged.  It turns out this is an issue with any error raised from C when it is called from the Perl code.  This has been fixed at 1359e2908c777ef7.

If this is a dev system and you can build from master then it would be good to see the trace output from a new backup.  We only need to see the log after backup remove starts, `INFO: remove expired backup ...`.

If you can't build from master then we'll need to wait for the next release to see exactly what is going on here, but a trace log might still be helpful.

**Comment by hunleyd** [comment]
this has stopped happening for me. I don't know if ExpanDrive updated itself or GDricve updated, but I can't repro anymore. So, as per convo on slack, closing. Will re-open if it resurfaces.
